spec/models/restaurant_spec.rb

```diff
@@ -1,5 +1,16 @@
 require 'rails_helper'
 
-# RSpec.describe Restaurant, type: :model do
-#   pending "add some examples to (or delete) #{__FILE__}"
-# end
+RSpec.describe Restaurant, type: :model do
```

Comment:
now, you have a documented set of behaviors that indicates who has the right to add/edit/delete items. You should continue with this convention as you build out more models